modules/Emails/language/en_us.lang.php

```diff
@@ -385,4 +385,28 @@
 
     'LBL_LIST_DATE_MODIFIED' => 'Last Modified',
     'LNK_IMPORT_CAMPAIGNS' => 'Import Campaign',
+    
+    // Email Validation Error messages. Typicaly for Email Validation:
+    'ERR_FIELD_FROM_IS_NOT_SET' => 'From field is not set.',
+    'ERR_FIELD_FROM_IS_EMPTY' => 'From field is empty.',
+    'ERR_FIELD_FROM_IS_INVALID' => 'From field is invalid.',
+    'ERR_FIELD_FROM_ADDR_IS_NOT_SET' => 'From address is not set.',
+    'ERR_FIELD_FROM_ADDR_IS_EMPTY' => 'From address is empty.',
+    'ERR_FIELD_FROM_ADDR_IS_INVALID' => 'From address is invalid.',
+    'ERR_FIELD_FROMNAME_IS_NOT_SET' => 'From-name is not set.',
+    'ERR_FIELD_FROMNAME_IS_EMPTY' => 'From-name is empty.',
+    'ERR_FIELD_FROMNAME_IS_INVALID' => 'From-name is invalid.',
```

Comment:
Why `From-name` in the key as a concept? But on the next strings its used `From name`.
For translations its better to have consistence and normal english words

Reply:
I agree with Horus but I do think the two words go together as a concept, and it might not be obvious unless you give the user some clue to that. I think the proper way to do it could be quote marks: `"From name" is invalid.`

Reply:
Redundant "from name" are in the php code (most of these "From name/address issues" by the inconsistent variable naming). Normally when I create an validation error message should determinate which data was wrong. At the moment in the code is redundancy. The string `"From name is not set"` already exists (see a few line bellow), however a differents php variable cause that error message. If I want to pointing to an error clearly I have to be clean with a uniquie error messages to make sense where the possible issue came from in code. Thats why I picked the "From name" and "From-name" string in the error message. If you have a better Idea, I open minded..